doc/source/whatsnew/v1.3.0.rst

```diff
@@ -277,10 +277,10 @@ As an example of this, given:
         unique
         original.dtype == unique.dtype
 
-Preserve dtypes in  :meth:`~pandas.DataFrame.combine_first`
+Preserve dtypes in  :meth:`DataFrame.combine_first`
```

Comment:
doesn't this warn (line lenght)?

Reply:
Thanks for catching this - I rebuilt the docs locally and got no warnings. CI is showing the same - not sure why that is.

Reply:
From reST docs (https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#sections):

> An underline/overline is a single repeated punctuation character that begins in column 1 and forms a line extending at least as far as the right edge of the title text. Specifically, an underline/overline character may be any non-alphanumeric printable 7-bit ASCII character 5. When an overline is used, the length and character used must match the underline.

Reply:
I think, we had some cases where I forgot to reduce the underline when changing release dates in previous whatsnew. didn't seem to do any harm, and rendered fine.